templates/admin/donator/show_donations.html.twig

```diff
@@ -4,7 +4,7 @@
     <ul>
         {% for donation in object.donations %}
             <li>
-                <a href="{{ path('admin_app_donation_show', { id: donation.id }) }}">Le {{ donation.createdAt|localizeddate('none', 'none', 'fr_FR', null, 'd MMM Y HH:MM') }} : {{ donation.amountInEuros|localizedcurrency('EUR') }}</a>
+                <a href="{{ path('admin_app_donation_show', { id: donation.id }) }}">Le {{ donation.createdAt|localizeddate('none', 'none', 'fr_FR', null, 'd MMM Y HH:mm') }} : {{ donation.amountInEuros|localizedcurrency('EUR') }}</a>
```

Comment:
👍 could you fix other occurrences of the same mistake, please? 
![image](https://user-images.githubusercontent.com/4582866/63857181-8a615e80-c9a3-11e9-8423-150abd7edad2.png)

Reply:
Yes I'm creating a new PR for that :ok_hand: